R/rlistings.R

```diff
@@ -31,10 +32,10 @@ setOldClass(c("MatrixPrintForm", "list"))
 #'
 #' dat <- ex_adae
 #'
-#' lsting <- as_listing(dat[1:25,], key_cols = c("USUBJID", "AESOC")) %>%
-#'     add_listing_col("AETOXGR") %>%
-#'     add_listing_col("BMRKR1", format = "xx.x") %>%
-#'     add_listing_col("AESER / AREL", fun = function(df) paste(df$AESER, df$AREL, sep = " / "))
+#' lsting <- as_listing(dat[1:25, ], key_cols = c("USUBJID", "AESOC")) %>%
```

Comment:
@cicdguy we already ran rlistings through the linter, why is this PR changing essentially every line of code, again?

Reply:
Also even if we are going to lint it (again) taht should be a separate PR shouldn't it?

Reply:
I don't know, @edelarua did it the last time around.

Reply:
We didn't use `styler` last time. Only corrected the lint.